packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import warning from 'warning';
+import {BreadcrumbEllipsis} from './BreadcrumbEllipsis';
+import {BreadcrumbItem, IBreadcrumbItem} from './BreadcrumbItem';
+
+interface IProps extends React.HTMLAttributes<HTMLOListElement> {
+	ellipsisBuffer?: number;
+
+	ellipsisProps?: Object;
+
+	items: Array<IBreadcrumbItem>;
+
+	spritemap: string;
+}
+
+const getBufferList = (
+	items: Array<IBreadcrumbItem>,
+	start: number,
+	end: number,
+	ellipsisProps: Object,
```

Comment:
instead of an explicit prop here, can we just use `...otherProps` since bufferList implies it is for an ellipsis

Reply:
`otherProps` is already being used on `<ol>` root